common/server-data/prismic.ts

```diff
@@ -25,7 +25,7 @@ export type ResultsLite = {
 export const defaultValue = {
   globalAlert: {
     data: {
-      isShown: null,
+      isShown: 'hide' as 'hide' | 'show',
```

Comment:
Can you explain the `'hide' as 'hide'` for my own knowledge gaining? :D

Reply:
If I give it the value 'hide' without the `as` there is a typescript error saying`Type 'string' is not assignable to type'"hide" | "show"'. So I had to tell it the value is one of those specific strings. It think `as const` would be better actually, then we wouldn't have to change this if we added more possible strings